package.json

```diff
@@ -4,20 +4,19 @@
   "description": "The command-line tool for building standalone Expo apps",
   "preferGlobal": true,
   "scripts": {
-    "start": "EXPENV_OVERRIDES=local-common,local expenv gulp",
+    "start": "EXPENV_OVERRIDES=local-common,local,local-queue expenv gulp",
     "start:ios": "PLATFORM=ios yarn start",
     "start:android": "PLATFORM=android yarn start",
-    "watch": "EXPENV_OVERRIDES=local-common,local expenv gulp watch:source",
+    "watch": "EXPENV_OVERRIDES=local-common,local,local-queue expenv gulp watch:source",
     "bin:turtle": "./bin/turtle.js",
     "lint": "eslint src",
     "prepare": "yarn build",
     "build": "gulp build:server",
     "build:production": "./scripts/build.sh",
     "build:android-tarball:local": "./scripts/android/buildAndroidTarballLocally.sh",
-    "init-workingdir:local": "./scripts/initLocalWorkingdir.sh",
-    "init-workingdir:remote": "yarn init-workingdir:remote-android && yarn init-workingdir:remote-ios",
```

Comment:
`scripts/build.sh` script uses `init-workingdir:remote`, so it has to be updated as well.